### ISSUE TITLE
conductive_blocks

### ISSUE THREAD

**Issue by twahm** [minetest-mods/minetest-mods.github.io#52]
This mod add a set of conductive blocks to the game [conductive_blocks]( https://github.com/twahm/conductive_blocks)

**Comment by sofar** [comment]
This is not necessarily a mod that is a good candidate for minetest-mods.

- it's only 1 day old, so it's definitely not "stable"
- no license
- no description.txt
- no hints in the readme that this goes with mesecons
- no forum post, so nobody knows it exists
- probably not in use anywhere yet
- somewhat esoteric as it doesn't offer any new functionality (it's all aestatics, essentially).

I'd say, come back in a few months or after a bunch of servers are running this mod. For now, I'll reject it as "too much WIP".

**Comment by twahm** [comment]
1. I upload the mods to github after finishing them, not when I am doing or testing them.
2. You are right you do not have a license.
3. It's the first mod that I created so I did not know there was a description.txt.
4. I would post that in the forum
5. I expected it to be approved to be published in the forum
6. Yes, nobody has it yet
7. It is not a new functionality I know, but it is a way of using mesecons without damaging what you are doing, when you build something, placing cables damages aesthetics or there is no space
